src/BuckarooClient.php

```diff
@@ -118,8 +118,7 @@ public function client()
      */
     private function getConfig(string $websiteKey, string $secretKey, string $mode = null): ?Config
     {
-        if($websiteKey && $secretKey)
-        {
```

Comment:
My personal preference is to have brackets on the next line. This way, it's easier to see where it starts.